include/wolfboot/wolfboot.h

```diff
@@ -257,16 +257,16 @@ extern "C" {
 #ifndef WOLFBOOT_FLAGS_INVERT
 #define IMG_STATE_NEW       0xFF
 #define IMG_STATE_UPDATING  0x70
-#define IMG_STATE_FINAL_FLAGS  0x30
+#define IMG_STATE_FINAL_SWAP 0x30
```

Comment:
This value cannot be renamed because it will break customers. Either use the old name or add a new one.